### PR TITLE
Spring security web update

### PR DESCRIPTION
**Pull request by adkumar1**

### CODE REVIEW

**Comment by Wulghash** [comment]
Just a simple library update

**Comment by Siegfriedk** [comment]
> Just a simple library update

Make sure to still review PRs properly. See the wrong version number @Wulghash

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [Unreleased]
 
 
+## [2.0.3] - 2023-04-24
```
Comment (Siegfriedk) [inline CHANGELOG.md:10]: version is wrong

**Review by Siegfriedk — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (adkumar1) [reply CHANGELOG.md]: corrected the version

**Pull request merged**